javascript/ql/lib/change-notes/2022-04-04-dataflow-models.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* Many library models have been rewritten to use dataflow nodes instead of the AST.
+  These changes include some renamings where the old name is still available as a deprecated feature.
```

Comment:
There are _some_ breaking changes, right? I think it's ok for a change like this, where it's practically unavoidable.

But if there are breaking changes, I think we should call out that some breaking changes have occurred, but in most cases, the AST-based predicates and classes are simply deprecated.

Reply:
> There are some breaking changes, right? 

Definitely yes, I hit a decent amount of compile-errors while doing this refactoring.   
I've changed the category to `breaking`, and reworded the change-note a bit.